.github/workflows/podman-test.yaml

```diff
@@ -0,0 +1,26 @@
+name: GitHub Actions Demo
```

Comment:
Could you please change the name of the Action, so it is easily findable in the list?